scalafix-sbt/src/main/scala-sbt-0.13/scalafix/internal/sbt/SBTCompat.scala

```diff
@@ -0,0 +1,11 @@
+package scalafix.internal.sbt
+
+object SBTCompat {
+  object CrossVersion {
+    def partialVersion(version: String): Option[(Long, Long)] = {
+      sbt.CrossVersion.partialVersion(version).map{ case (major, minor) =>
```

Comment:
This might be a good contribution to https://github.com/dwijnand/sbt-compat/blob/master/src/main/scala-sbt-0.13/compat.scala

Reply:
cc/ @dwijnand

Reply:
Yup! Please send a PR.